packages/fx-core/src/plugins/resource/aad/index.ts

```diff
@@ -89,6 +89,8 @@ export class AadAppForTeamsPlugin implements Plugin {
         errorMessage += ` Detailed error: ${e.innerError.message}.`;
         if (e.innerError.response?.data?.errorMessage) {
           errorMessage += ` Reason: ${e.innerError.response?.data?.errorMessage}`;
+        } else if (e.innerError.response?.data?.error?.message) {
```

Comment:
add some comments to tell where does the error come from? E.g. this line reads error returned by Graph API
This could help others understand the logic, since we get the error information from multiple sources

Reply:
Fixed. Thank you for pointing that out.